packages/cli/src/commands/login/index.ts

```diff
@@ -68,14 +71,24 @@ export default async function login(client: Client): Promise<number> {
     return result;
   }
 
-  const isNewLogin = !client.authConfig.token;
-
   // Save the user's authentication token to the configuration file.
   client.authConfig.token = result.token;
 
-  // If we have a new login, update `currentTeam`
-  if (isNewLogin) {
-    await updateCurrentTeamAfterLogin(client, output, result.teamId);
+  // Save the user's team (possibly `undefined`) to the global configuration file.
+  client.config.currentTeam = result.teamId;
+
+  // Delete undefined `currentTeam` key to avoid writing key with no value to JSON file.
+  if (!client.config.currentTeam === undefined) {
```

Comment:
The `!` seems unnecessary?

```suggestion
  if (client.config.currentTeam === undefined) {
```